save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -47,8 +47,9 @@ class Save(
     @Suppress("WHEN_WITHOUT_ELSE")  // TestResult is a sealed class
     private fun handleResult(testResult: TestResult) {
         val status = testResult.status
+        val passMessage = "Test on resources [${testResult.resources}] has completed successfully"
```

Comment:
May be we can move initialization of this message into the block of Pass?
```kotlin
is Pass -> {
    val passMessage = "Test on resources [${testResult.resources}] has completed successfully"
    status.message?.let { logDebug("$passMessage. $it") } ?: logDebug(passMessage)
}
```

Reply:
Yes)